code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -50,9 +50,17 @@
 	message = "screams!"
 	message_mime = "acts out a scream!"
 	emote_type = EMOTE_AUDIBLE
-	only_forced_audio = TRUE
 	vary = TRUE
 
+/datum/emote/living/carbon/human/scream/select_message_type(mob/user, msg, intentional)
+	. = ..()
+	var/mob/living/carbon/human/human = user
+	if(human.dna.species?.scream_verb)
+		if(human.mind?.miming)
+			return "[human.dna.species?.scream_verb] silently!"
+		else
+			return "[human.dna.species?.scream_verb]!"
```

Comment:
`human.dna` should *always* have a `species`, so these checks shouldn't be necessary.
```suggestion
	if(human.dna.species.scream_verb)
		if(human.mind?.miming)
			return "[human.dna.species.scream_verb] silently!"
		else
			return "[human.dna.species.scream_verb]!"
```